usb_autorunner_profiles/metrics/run.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+kubectl="/usr/local/bin/k3s kubectl"
+
+AUTORUNNER_WORKDIR=/opt/usb-autorunner/workdir
+PROMETHEUS_METRICS_JOB_NAME=prometheus-snapshot-backup
+
+# Retrieve Docker registry IP address
+echo "🗂  Retrieve Docker registry IP."
+REGISTRY_IP=${REGISTRY_IP:-10.0.90.99}
+
+# Sync images to registry
+echo "⚙️  Upload container images to the registry at $REGISTRY_IP..."
+cd $AUTORUNNER_WORKDIR/images/docker.io && skopeo sync --scoped --dest-tls-verify=false --src dir --dest docker ./  $REGISTRY_IP
+
+# Get USB mount point
+usb_mount_point=`grep "mount_point" /etc/usb-autorunner/usbinfo | cut -d'=' -f2 | tr -d '"'`
+metrics_folder=${usb_mount_point}/metrics-$(date +'%Y-%m-%d_%H-%M')/
+echo "ℹ️ Archives will be saved in '${metrics_folder}'"
+mkdir -p $metrics_folder
+
+echo "⚙️  Delete old backup jobs"
+$kubectl delete job -n monitoring -l app=metrics-backup -n monitoring --ignore-not-found=true
```

Comment:
Are you providing the `-n monitoring` twice intentionally?

Reply:
No, will update it.